jekyll-assets/_layouts/docs.html

```diff
@@ -1,26 +1,181 @@
 <!doctype html>
 <html lang="en">
   <head>
-    {% include head.html %} 
+    {% include head.html %}
   </head>
-  <body>
-    {% include header.html %}      
-    {% include toptitle.html %}
-    {% include tabs.html %}
-
-    <div id="container">
-      {% include mobile_nav.html %}
-      {% include nav.html %}
-
-      <section id="content">
-        <h1>{{ page.sub_title | markdownify | remove: '<p>' | remove: '</p>'}}</h1>
-        {{ content }}
-      </section>
+  <body onload="makeToc()">
+    <script type="text/javascript">initTheme()</script>
+    {% include header.html %}
+    <input type="checkbox" id="mobile-toggle" />
+    <div id="docs-content">
+      <div id="toc-container-container">
```

Comment:
Are we going to be using Docker? :rofl: :ship: :ship:

Reply:
yo dawg, i heard you like containers, so i put a container in your container